nomad/structs/consul.go

```diff
@@ -26,6 +26,12 @@ const (
 	ConsulWorkloadsDefaultAuthMethodName = "nomad-workloads"
 )
 
+type ConsulACLToken struct {
+	Cluster  string
+	TokenID  string
+	ACLToken string
+}
```

Comment:
This only exists in the client state and never gets sent over the wire, so I'd probably put this struct in `client/structs`